src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/StraxCoinstakeRule.cs

```diff
@@ -17,7 +16,18 @@ public class StraxCoinstakeRule : PartialValidationConsensusRule
         // The actual percentage of the reward sent to this script is checked within the coinview rule.
         // This is an anyone-can-spend scriptPubKey as no signature is required for the redeem script to be valid.
         // Recall that a scriptPubKey is not network-specific, only the address format that it translates into would depend on the version bytes etc. defined by the network.
-        public static readonly Script CirrusRewardScript = new Script(new List<Op>() { OpcodeType.OP_TRUE }).PaymentScript;
+
+        // The redeem script is defined first (and separately) because it is needed for claiming the reward.
+        // It is not the scriptPubKey that must appear in the reward transaction output.
+        public static readonly Script CirrusRewardScriptRedeem = new Script(new List<Op>() { OpcodeType.OP_TRUE });
+
+        // This payment script is what must actually be checked against in the consensus rule i.e. the reward transaction has this as an output's scriptPubKey.
+        public static readonly Script CirrusRewardScript = CirrusRewardScriptRedeem.PaymentScript;
+
+        // This is not used within consensus, but it makes sense to keep the value close to the other script definitions so that it isn't buried inside the reward claimer.
+        // TODO: Replace this & its script with a vanity/burn address with unknowable private key
+        public static readonly string CirrusDummyAddress = "CPqxvnzfXngDi75xBJKqi4e6YrFsinrJka";
+        public static readonly Script CirrusTransactionTag = new Script("OP_DUP OP_HASH160 50f5ab5aef22f0fd6699bb60ce39fd7764456e5f OP_EQUALVERIFY OP_CHECKSIG");
```

Comment:
```suggestion
        public static readonly Script CirrusTransactionTag = new Script("OP_RETURN 43507178766e7a66586e674469373578424a4b716934653659724673696e724a6b61");
```

This was meant to be changed to the script format of a cross chain transfer